scripts/templates/grafana-dashboards/dm-apps.json

```diff
@@ -0,0 +1,747 @@
+{
+  "title": "DM-Apps",
+  "tags": [],
+  "style": "dark",
+  "timezone": "browser",
+  "editable": true,
```

Comment:
Ok, just to point out, this only changes whether or not you can change the rows. You will still be able to move graphs around or change what is displayed in them, just you'll not be able to add and change the rows.